addons/website/views/snippets/snippets.xml

```diff
@@ -593,7 +593,7 @@
     <div data-js="layout_column"
         data-selector="section, section.s_carousel_wrapper .carousel-item"
         data-target="> *:has(> .row), > .s_allow_columns"
-        data-exclude=".s_masonry_block, .s_features_grid, .s_media_list, .s_showcase, .s_table_of_content, .s_process_steps, .s_image_gallery">
+        data-exclude=".s_masonry_block, .s_features_grid, .s_media_list, .s_showcase, .s_table_of_content, .s_process_steps, .s_image_gallery, .s_dynamic">
```

Comment:
Not 100% sure but my guess is that before this PR the layout was not made in a way that it would activate the layout option ? The `.row` is now a direct child of the container, it was not the case before. Made a test and it looks like it comes from that.

Before:
<img width="247" alt="Screenshot 2024-05-07 at 14 24 16" src="https://github.com/odoo/odoo/assets/110090660/397c6d11-d134-4d2f-aaa1-8efbded465e7">

After this PR: 
<img width="290" alt="Screenshot 2024-05-07 at 14 28 27" src="https://github.com/odoo/odoo/assets/110090660/21bc979e-2950-4b14-8933-be3ae2069907">

Reply:
I see :+1: